Source/PeleLMeX_PatchFlowVariables.H

```diff
@@ -0,0 +1,10 @@
+#ifndef PELELM_PATCHFLOWVARIABLES_H
+#define PELELM_PATCHFLOWVARIABLES_H
+
+class PeleLM;
+
+void patchFlowVariables(
```

Comment:
How about making this a member of the PeleLM class (move the declaration to PeleLMeX.H)?

Reply:
Let's consider this for a future PR if it's not mandatory now?

Reply:
Sure, sounds good.